src/codegen/ruby/def/field.rb

```diff
@@ -23,19 +23,19 @@ def java_name
   
     prefix + name.tr('?$', '').split('_').collect(&:capitalize).join
   end
-  
+
```

Comment:
What's with all these whitespace changes?

Reply:
I actually have no idea. They don't affect anything, I think it was just my IDE not wanting it there.